src/tyr/isochrone_serializer.cc

```diff
@@ -265,17 +351,39 @@ std::string serializeIsochronePbf(Api& request,
 namespace valhalla {
 namespace tyr {
 
+#ifdef ENABLE_GDAL
 std::string serializeIsochrones(Api& request,
                                 std::vector<midgard::GriddedData<2>::contour_interval_t>& intervals,
-                                midgard::GriddedData<2>::contours_t& contours,
-                                bool polygons,
-                                bool show_locations) {
+                                const std::shared_ptr<const midgard::GriddedData<2>>& isogrid,
+                                valhalla::thor::geotiff_driver_t geotiff_driver) {
+#else
```

Comment:
this looks like it'd be better served with an overload instead of `#ifdef`ing in the arguments

Reply:
then that type would need to made available though even though `ENABLE_GDAL` is off?